package.json

```diff
@@ -91,6 +91,6 @@
     "eslint": ">=9 <10"
   },
   "engines": {
-    "node": ">=20.17"
+    "node": ">=22.11"
```

Comment:
If we're bumping engine requirement, this should be a MAJOR label.   Maybe leave it for a different PR and keep it at 20.x